src/js/helpers/JSONPUtil.js

```diff
@@ -0,0 +1,44 @@
+import Q from "q";
+
+const JSONPUtil = {
+  /**
+   * Request JSONP data
+   *
+   * @todo: write test to verify this util is working properly
```

Comment:
we can probably revisit this once we make progress on https://github.com/mesosphere/marathon/issues/2099